skimage/transform/_geometric.py

```diff
@@ -765,8 +765,8 @@ class AffineTransform(ProjectiveTransform):
     ----------
     matrix : (3, 3) array, optional
         Homogeneous transformation matrix.
-    scale : (sx, sy) as array, list or tuple, optional
-        Scale factors.
+    scale : {s as float or (sx, sy) as array, list or tuple}, optional
+        Scale factor(s). If a single value, it will be assigned to both sx and sy.
```

Comment:
Line too long for PEP8

```suggestion
        Scale factor(s). If a single value, it will be assigned to both
        sx and sy.
```